akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala

```diff
@@ -157,6 +152,13 @@ abstract class ActorSystem[-T] extends ActorRef[T] with Extensions { this: Inter
    */
   def receptionist: ActorRef[Receptionist.Command] =
     Receptionist(this).ref
+
+  /**
+   * Return a reference to this system’s [[akka.actor.typed.eventstream.EventStream]].
+   */
```

Comment:
I think something like the untyped event stream scaladoc is better: `Main event bus of this actor system, used for example for logging.`

`akka.actor.typed.eventstream.EventStream` is internal so not very useful to link there, perhaps instead provide a link to `akka.actor.typed.eventstream.EventStream.Subscribe`